README.md

```diff
@@ -1,3 +1,104 @@
-# secure-api-gateway-releases
-This repo holds the versioned helm and kustomize objects to be able to deploy the third party sources and secure-api-gateway
+# Secure API Gateway - Releases
 
+## Prerequisites
```

Comment:
Could do with an Overview section here. It should answer the following questions. 

Why does this repository exist?
What was your motivation?
Why did you build this project?
What problem does it solve?

Reply:
Added in generic introduction & repo specific overview